src/utils/api-request.ts

```diff
@@ -108,6 +109,10 @@ class DefaultHttpResponse implements HttpResponse {
       `request: "${ this.request }."`,
     );
   }
+
+  get isJson(): boolean {
```

Comment:
If you need a method to check if a response is valid, I think it is better to give it another name. `isValid()`. or something like that. Also a `isSomething()` should be a method. If you need a property for this, call it something like `valid`.
Note JSCore style guide:
`Avoid using the 'is' prefix for boolean properties unless it's unclear from context that the property is a boolean.`

Reply:
It's not a matter of validity. It's about whether the response is json or not. We have endpoints (e.g. Metadata server), where the valid response is plain text. It also seems at least some of our backend endpoints return html error messages. So we need a way to easily differentiate between json and non-json responses, so the client code can handle them easily.